src/features/transfer/TransferTokenForm.tsx

```diff
@@ -430,11 +443,26 @@ function ReviewDetails({ visible, tokenRoutes }: { visible: boolean; tokenRoutes
           <div>
             <h4>{`Transaction${isApproveRequired ? ' 2' : ''}: Transfer Remote`}</h4>
             <div className="mt-1.5 ml-1.5 pl-2 border-l border-gray-300 space-y-1.5 text-xs">
-              <p>{`Remote Token: ${route?.destRouterAddress}`}</p>
+              <p className="flex">
+                <span className="min-w-[7rem]">Remote Token</span>
+                <span>{route?.destRouterAddress}</span>
+              </p>
               {isNft ? (
-                <p>{`Token ID: ${sendValue}`}</p>
+                <p className="flex">
+                  <span className="min-w-[7rem]">Token ID</span>
+                  <span>{sendValue}</span>
+                </p>
               ) : (
-                <p>{`Amount (${originUnitName}): ${sendValue}`}</p>
+                <>
+                  <p className="flex">
+                    <span className="min-w-[7rem]">{`Amount (${originUnitName})`}</span>
+                    <span>{sendValue}</span>
+                  </p>
+                  <p className="flex">
+                    <span className="min-w-[7rem]">{`Interchain Gas (${originUnitName})`}</span>
```

Comment:
If we're bridging an ERC-20 from Ethereum, I don't think it's super clear that the interchain gas is denominated in ETH but the "Amount" on line 458 is denominated in the ERC-20 token

Reply:
I'll add the symbol names to the amounts